docs/builder.rst

```diff
@@ -49,7 +49,7 @@ is processed.
    :scale: 75 %
 
 
-For every discovered buildspecs, buildtest will validate the buildspecs in the :ref:`parse stage <_parse_buildspecs>` to
+For every discovered buildspecs, buildtest will validate the buildspecs in the :ref:`parse stage <parse_buildspecs>` to
```

Comment:
I believe these should be `buildspec` instead of `buildspecs` since you're talking about the singular here.